src/main/java/frc/robot/subsystems/climb/ClimbIOSim.java

```diff
@@ -0,0 +1,79 @@
+package frc.robot.subsystems.climb;
+
+import static edu.wpi.first.units.Units.*;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.units.measure.MutAngle;
+import edu.wpi.first.units.measure.MutVoltage;
+import edu.wpi.first.units.measure.Voltage;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class ClimbIOSim implements ClimbIO {
+
+    private SingleJointedArmSim climb =
+            new SingleJointedArmSim(
+                    DCMotor.getKrakenX60(2),
+                    75,
+                    SingleJointedArmSim.estimateMOI(0.5, 0.5),
+                    0.5,
+                    0,
+                    2 * Math.PI,
+                    false,
+                    0,
+                    0,
+                    0);
```

Comment:
You might need to add in a non-zero value for the second to last argument, like is done in the example code: https://github.com/wpilibsuite/allwpilib/blob/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/armsimulation/subsystems/Arm.java